keyboards/malevolti/lyra/lyra.c

```diff
@@ -0,0 +1,89 @@
+ /* Copyright 2021 Domanic Calleja
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+#include "lyra.h"
+
+#ifdef OLED_DRIVER_ENABLE
+
+// 'lyralogooled', 32x128px
+__attribute__((weak)) oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    if (is_keyboard_master()) {
+        return OLED_ROTATION_270;
+    }
+    return rotation;
+}
+
+__attribute__((weak)) void oled_task_user(void) {
+    if (is_keyboard_master()) {
+        oled_write_P(PSTR("      "), false);
+#    ifdef WPM_ENABLE
+      uint8_t n = get_current_wpm();
+        char wpm_counter[4];
+        wpm_counter[3] = '\0';
+        wpm_counter[2] = '0' + n % 10;
+        wpm_counter[1] = (n /= 10) % 10 ? '0' + (n) % 10 : (n / 10) % 10 ? '0' : ' ';
+        wpm_counter[0] = n / 10 ? '0' + n / 10 : ' ';
+        oled_write(wpm_counter, false);
```

Comment:
This can now be condensed to: 
```suggestion
        oled_write(get_u8_str(get_current_wpm(), ' '), false);
```